contents/tutorials/react-analytics.md

```diff
@@ -0,0 +1,244 @@
+
+```js file=src/AboutPage.js
+import React from 'react';
+
+function AboutPage() {
+  return <h1>About Page</h1>;
+}
+
+export default AboutPage;
+```
+
+Next we set up the routing. First install React Router in your project:
+
+```bash
+npm install react-router-dom
+```
+
+Then, set up the router by replacing the code in `App.js` with the following:
+
+```js file=App.js
+import React from 'react';
+import { BrowserRouter as Router, Route, Link, Routes } from 'react-router-dom';
+import HomePage from './HomePage';
+import AboutPage from './AboutPage'; 
+
+function App() {
+  return (
+    <Router>
+      <div>
+        <nav>
+          <ul>
+            <li>
+              <Link to="/">Home</Link>
+            </li>
+            <li>
+              <Link to="/about">About</Link>
+            </li>
+          </ul>
+        </nav>
+
+        <Routes> 
+          <Route path="/about" element={<AboutPage />} />
+          <Route path="/" element={<HomePage />} />
+        </Routes>
+      </div>
+    </Router>
+  );
+}
+
+export default App;
+```
+
+The basic setup is now complete. Run `npm start` to see your app in action.
+
+![Basic React app](../images/tutorials/react-analytics/basic-app.png)
+
+## Adding PostHog
+
+With our app set up, it’s time to install and set up PostHog. If you don't have a PostHog instance, you can [sign up for free](https://us.posthog.com/signup). 
+
+First install `posthog-js`:
+
+```bash
+npm install posthog-js
+```
+
+Next, import PostHog into `src/index.js` and set up a client using your project API key and instance address. You can find these in [your project settings](https://app.posthog.com/project/settings). Then we wrap our app with `PostHogProvider` to access PostHog in any component.
+
+```js file=src/index.js
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import posthog from 'posthog-js'
+import { PostHogProvider } from 'posthog-js/react'
+
+posthog.init('<ph_project_api_key>', {
+  api_host: '<ph_instance_address>',
+})
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <PostHogProvider client={posthog}>
+      <App />
+    </PostHogProvider>
+  </React.StrictMode>
+);
+```
+
+Once you’ve done this, reload your app and click the links a few times. You should see events appearing in the [PostHog events explorer](https://us.posthog.com/events).
+
+<ProductScreenshot
+  imageLight={EventsLight} 
+  imageDark={EventsDark} 
+  alt="Events in PostHog" 
+  classes="rounded"
+/>
+
+## Capturing pageviews
+
+You might notice that moving between pages only captures a single pageview event. This is because PostHog only captures pageview events when a [page load](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) is fired. Since React creates a single-page app, this only happens once, and the React router handles subsequent page changes.
```

Comment:
```suggestion
You might notice that moving between pages only captures a single pageview event. This is because PostHog only captures pageview events when a [page load](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) is fired. Since React creates a single-page app, this only happens once and the React router handles subsequent page changes.
```